### PR TITLE
Add new repo leda-contrib-vscode-extensions

### PR DESCRIPTION
**Pull request by heurtematte**
Related to https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3412

### CODE REVIEW

**Comment by heurtematte** [comment]
@mikehaller I took some common configurations from other repositories. Tell me if it fits.

**Comment by netomi** [comment]
GitHub added the secret scanning features only a few months ago, thats why many repos dont have it enabled yet, but we want to enable that for as many repos as possible. For new repos I would always enable it, for all existing repos I would make a PR to enable it for them in one go.

**Comment by heurtematte** [comment]
`secret_scanning` and `secret_scanning_push_protection` are enabled.

File: otterdog/eclipse-leda.jsonnet [hunk 1]
```diff
@@ -89,6 +89,12 @@ orgs.newOrg('eclipse-leda') {
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
     },
+    orgs.newRepo('leda-contrib-vscode-extensions') {
+      allow_update_branch: false,
+      secret_scanning: "enabled",
```
Comment (netomi) [inline otterdog/eclipse-leda.jsonnet]: you can remove these lines wrt secret scanning, they are already defined in the default.
See the diff to the canonical version, its hightlighted that setting them again is redundant.

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
repo has been added.

**Pull request merged**